restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/CallRegexNullSrcOrgTest.java

```diff
@@ -0,0 +1,1649 @@
+            restcommContact = "127.0.0.1:" + restcommPort;
+        }
+        if (System.getProperty("arquillian_http_port") != null) {
+            restcommHTTPPort = Integer.valueOf(System.getProperty("arquillian_http_port"));
+        }
+    }
+
+    @Before
+    public void before() throws Exception {
+        bobSipStack = tool1.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", bobPort, restcommContact);
+        bobPhone = bobSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, restcommPort, bobContact);
+
+        aliceSipStack = tool2.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", alicePort, restcommContact);
+        alicePhone = aliceSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, restcommPort, aliceContact);
+
+        henriqueSipStack = tool3.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", henriquePort, restcommContact);
+        henriquePhone = henriqueSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, restcommPort, henriqueContact);
+
+        georgeSipStack = tool4.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", georgePort, restcommContact);
+        georgePhone = georgeSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, restcommPort, georgeContact);
+
+        subAccountClientSipStack = tool5.initializeSipStack(SipStack.PROTOCOL_UDP, "127.0.0.1", subAccountPort, restcommContact);
+        subAccountClientPhone = subAccountClientSipStack.createSipPhone("127.0.0.1", SipStack.PROTOCOL_UDP, restcommPort, subAccountClientContact);
+    }
+
+    @After
+    public void after() throws Exception {
+        if (bobPhone != null) {
+            bobPhone.dispose();
+        }
+        if (bobSipStack != null) {
+            bobSipStack.dispose();
+        }
+
+        if (aliceSipStack != null) {
+            aliceSipStack.dispose();
+        }
+        if (alicePhone != null) {
+            alicePhone.dispose();
+        }
+
+        if (henriqueSipStack != null) {
+            henriqueSipStack.dispose();
+        }
+        if (henriquePhone != null) {
+            henriquePhone.dispose();
+        }
+
+        if (georgePhone != null) {
+            georgePhone.dispose();
+        }
+        if (georgeSipStack != null) {
+            georgeSipStack.dispose();
+        }
+
+        if (subAccountClientPhone != null) {
+            subAccountClientPhone.dispose();
+        }
+        if (subAccountClientSipStack != null) {
+            subAccountClientSipStack.dispose();
+        }
+        Thread.sleep(1000);
+        wireMockRule.resetRequests();
+        Thread.sleep(4000);
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        System.gc();
+        System.out.println("System.gc() run");
+    }
+
+    private String dialAliceRcml = "<Response><Dial><Client>alice</Client></Dial></Response>";
+
+    @Test
+    public void testDialClientAlice7777() throws ParseException, InterruptedException, MalformedURLException {
```

Comment:
are we missign feature categories in these new test cases?

Reply:
I will annotate the whole test class with org.restcomm.connect.commons.annotations.FeatureExpTests